internal/log/log.go

```diff
@@ -67,3 +70,11 @@ type Logger interface {
 	Info(context.Context, string, ...Value)
 	Error(context.Context, string, ...Value)
 }
+
+func Trace() string {
+	pc := make([]uintptr, 15)
+	n := runtime.Callers(2, pc)
+	frames := runtime.CallersFrames(pc[:n])
+	frame, _ := frames.Next()
+	return frame.Function
+}
```

Comment:
This is really really good. Thanks for this!

Reply:
Thank you :)